### ISSUE TITLE
.c shouldn't scrape DuckDuckGo HTML

### ISSUE THREAD

**Issue by auscompgeek** [myano/jenni#161]
The DuckDuckGo API returns calculations perfectly fine...

Moreover, why is it scraping HTML before trying the API?

**Comment by auscompgeek** [comment]
A side effect of attempting to scrape HTML before using the API: if BeautifulSoup isn't installed, it borks completely.

**Comment by kaneda** [comment]
@auscompgeek this is explicitly mentioned in the readme: https://github.com/myano/jenni

**Comment by auscompgeek** [comment]
@kaneda Yes, but it causes an exception, so it doesn't even attempt to get a result from the DDG API or from Wolfram|Alpha.

**Comment by myano** [comment]
The "HTML" is scraped first before the API, because a) many times the "HTML" pages has a simpler answer and b) sometimes it is faster. I believe there was another reason of why I originally did it that way, but I can't remember any other reasons as of now.

**Comment by myano** [comment]
Personally, I really wish the google results for .c were more reliable in jenni. They seem/work more reliably when done in a browser.
